pandas/core/window/ewm.py

```diff
@@ -71,30 +72,48 @@ class EWM(_Rolling):
 
     Notes
     -----
-    Exactly one of center of mass, span, half-life, and alpha must be provided.
-    Allowed values and relationship between the parameters are specified in the
-    parameter descriptions above; see the link at the end of this section for
-    a detailed explanation.
-
-    When adjust is True (default), weighted averages are calculated using
-    weights (1-alpha)**(n-1), (1-alpha)**(n-2), ..., 1-alpha, 1.
-
-    When adjust is False, weighted averages are calculated recursively as:
-       weighted_average[0] = arg[0];
-       weighted_average[i] = (1-alpha)*weighted_average[i-1] + alpha*arg[i].
-
-    When ignore_na is False (default), weights are based on absolute positions.
-    For example, the weights of x and y used in calculating the final weighted
-    average of [x, None, y] are (1-alpha)**2 and 1 (if adjust is True), and
-    (1-alpha)**2 and alpha (if adjust is False).
-
-    When ignore_na is True (reproducing pre-0.15.0 behavior), weights are based
-    on relative positions. For example, the weights of x and y used in
-    calculating the final weighted average of [x, None, y] are 1-alpha and 1
-    (if adjust is True), and 1-alpha and alpha (if adjust is False).
-
-    More details can be found at
-    https://pandas.pydata.org/pandas-docs/stable/user_guide/computation.html#exponentially-weighted-windows
+    Exactly one paramter: ``com``, ``span``, ``halflife``, or ``alpha``
```

Comment:
I think this should be part of the extended summary, not the notes. And I think the extended summary should provide information on what it does. It's not a blocker to get this merged, but there is a lot of content in this page, but reading it doesn't really explain what this functionality does IMO.

Reply:
You mean after "Provide exponential weighted functions."?

I agree that `Exactly one parameter: ``com``, ``span``, ``halflife``, or ``alpha``
must be provided.` could be added there. Should I do that and then delete the first paragraph from the notes?

Reply:
Yes, that's correct. You have information on the different sections, and what should be in each at: https://pandas.pydata.org/pandas-docs/stable/development/contributing_docstring.html